bip-0131.mediawiki

```diff
@@ -77,6 +77,26 @@ the user when their wallet contains many UTXOs that qualify it to benefit from
 a coalescing transaction. Wallets should not simply replace non-coalescing transactions
 with coalescing transactions in all instances.
 
+== Isn't this BIP bad because it encourage address re-use? ==
+
+Address re-use comes in two forms: re-use by the ''sender'', and re-use by the ''receiver''.
+
+Re-use by the sender is basically using the same address for the change output. This is generally considered bad
+since people looking through your transaction history can determine who you do business with. When
+you generate a new address for every change, your privacy is conserved as it is impossible to know which
+output is a recipient, and which output is the change output. This BIP has '''no effect''' on re-use
+by the sender.
+
+On the other hand, address re-use by the ''receiver'' occurs under completely different circumstances.
+When you publish an address and have multiple people send to that address, you are engaging in address re-use
+from the receiver. This activity has historically been considered bad because it leads to re-using a private key.
+When you re-use a private key too many times, you run the risk of an attacker performing statistical analysis
+on the multiple signatures, which can lead to an attacker finding out your private key.
```

Comment:
> By the way, if I re-use my address, that has no effect on anyone but myself. Other people are not affected at all.

I disagree with that statement. You, and you alone, reusing your address simplifies linking other people's addresses together. Due to Bitcoin's lack of full anonymity, privacy provided by the system is a public good, that must be treasured by everyone equally to be preserved.

> If somebody wants to give up some privacy for a massive decrease in transaction fees, then they should be able to do that. 

As a result, I disagree with that as well. We should aim for a design that does not allow one person to get a monetary benefit from hurting the privacy of others in the system.